sql/upgrade/sl3.0.sql

```diff
@@ -2,7 +2,7 @@
 
 -- With help of a few conditional statements handled by the Template toolkit,
 -- this migration file handles migration from all SQL-Ledger version up to 3.0
--- to all Ledgersmb up to 1.6
+-- to all Ledgersmb up to 1.10
```

Comment:
The commit message doesn't explain why we the migration target moves from 1.6 to 1.10; the idea was to have all migrations to a single history point, because that nicely fixes at least one side of the migration schema. At the time it was logical to choose 1.6.

Concluding: I think it's important to mention in the commit message why this shift is important.

Reply:
Reverted, 1.6 was indeed the proper endpoint.

Reply:
Ok. Where is this reversal?

Reply:
> Reverted, 1.6 was indeed the proper endpoint.

1.6 doesn't have `amount_tc` and `amount_bc`, yet your migration script uses these. So the target is *not* 1.6?

Reply:
SL migrations have been set to 1.6+mc since Dec 2017